guides/common/assembly_using-ansible-roles.adoc

```diff
@@ -5,6 +5,8 @@ ifdef::context[:parent-context: {context}]
 = Using Ansible Roles to Automate Repetitive Tasks on {Project} Hosts
 
 include::modules/proc_assigning-ansible-roles-to-an-existing-host.adoc[leveloffset=+1]
+include::modules/proc_removing-ansible-roles-from-a-host.adoc[leveloffset=+1]
+include::modules/proc_changing-the-order-of-ansible-roles.adoc[leveloffset=+1]
```

Comment:
We generally have a blank line between two includes if they aren't snippets. I guess your PR is fine because it already was like this. #followuptask